ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -0,0 +1,674 @@
+    },
+    dataDictionary: {
+      id: 'dataDictionary',
+      disabled: false,
+      faIcon: faBook,
+      label: 'Data Dictionary Icon',
+      showIcon: () => true,
+      style: { color: colors.white, fontSize: '20px', marginTop: '5px' },
+      tooltip: 'Data Dictionary',
+      hasContent: false,
+    },
+    annotations: {
+      id: 'annotations',
+      disabled: false,
+      faIcon: faEdit,
+      label: 'Annotations Icon',
+      showIcon: () => pageKey === 'reviewParticipantDetail',
+      style: { fontSize: '20px', marginLeft: '3px' },
+      tooltip: 'Annotations',
+      hasContent: true,
+    },
+    runtime: {
+      id: 'runtime',
+      disabled: !!runtimeStore.loadingError,
+      faIcon: null,
+      label: 'Cloud Icon',
+      showIcon: () => true,
+      style: { height: '22px', width: '22px' },
+      tooltip: runtimeTooltip(
+        'Cloud Analysis Environment',
+        runtimeStore.loadingError
+      ),
+      hasContent: true,
+    },
+    terminal: {
+      id: 'terminal',
+      disabled: !!runtimeStore.loadingError,
+      faIcon: faTerminal,
+      label: 'Terminal Icon',
+      showIcon: () => true,
+      style: { height: '22px', width: '22px' },
+      tooltip: runtimeTooltip(
+        'Cloud Analysis Terminal',
+        runtimeStore.loadingError
+      ),
+      hasContent: false,
+    },
+    genomicExtractions: {
+      id: 'genomicExtractions',
+      disabled: false,
+      faIcon: faDna,
+      label: 'Genomic Extraction',
+      showIcon: () => true,
+      // position: absolute is so the status icon won't push the DNA icon to the left.
+      style: {
+        height: '22px',
+        width: '22px',
+        marginTop: '0.25rem',
+        position: 'absolute',
+      } as CSSProperties,
+      tooltip: 'Genomic Extraction History',
+      hasContent: true,
+    },
+  };
+
+  return config[iconId];
+};
+
+interface HelpSidebarIconsProps extends IconConfigProps {
+  workspace: WorkspaceData;
+  cdrVersionTiersResponse: CdrVersionTiersResponse;
+  compoundRuntimeOps: CompoundRuntimeOpStore;
+  genomicExtractionJobs: GenomicExtractionJob[];
+  activeIcon: string;
+  onIconClick: (icon: IconConfig) => void;
+}
+export const HelpSidebarIcons = (props: HelpSidebarIconsProps) => {
+  const { workspace, cdrVersionTiersResponse, activeIcon, onIconClick } = props;
+
+  const defaultIcons: SidebarIconId[] = [
+    'criteria',
+    'concept',
+    'help',
+    'notebooksHelp',
+    'dataDictionary',
+    'annotations',
+  ];
+  const keys: SidebarIconId[] = defaultIcons.filter((key) =>
+    iconConfig(key, props).showIcon()
+  );
+
+  if (WorkspacePermissionsUtil.canWrite(workspace.accessLevel)) {
+    keys.push('runtime', 'terminal');
+  }
+
+  if (
+    serverConfigStore.get().config.enableGenomicExtraction &&
+    getCdrVersion(workspace, cdrVersionTiersResponse).hasWgsData
+  ) {
+    keys.push('genomicExtractions');
+  }
+
+  const icons = keys.map((key) => iconConfig(key, props));
+
+  return (
+    <>
```

Comment:
IS <> like React.Fragment?

Reply:
yes!  It's the Short Syntax described here: https://reactjs.org/docs/fragments.html